chrome/script.browserify.js

```diff
@@ -159,16 +173,67 @@ function getLoginData() {
   );
 }
 
+function usernameToClipboard() {
+  var username = this.split("/");
```

Comment:
I don't store username as a file name, username should come from the host app, just like you get password below.